src/components/Auth/Login/Login.react.js

```diff
@@ -311,6 +317,18 @@ class Login extends Component {
 									labelColor="#fff" />
 							</Link>
 						</div>
+						<h4 style={{
+							margin: '8px 0'
+						}}><Translate text="OR"/></h4>
+                                <FacebookLogin socialId="2002829926655326"
```

Comment:
Is this socialId same for everyone?

Reply:
@anshumanv , this socialId was used just for the purpose of testing,another id through fossasia's credentials should be used

Reply:
Okay, so fossasia's acc need to create an OAuth app and use their ID. :+1: